src/qt/veil/forms/receivewidget.ui

```diff
@@ -139,7 +140,11 @@ font-size:16px;</string>
            </property>
            <property name="text">
             <string>A sender can copy the address or scan the 
+<<<<<<< HEAD
```

Comment:
Fix this.

Reply:
Keep in mind I use the Mac OS X GitHub Desktop app, and have very minimal experience. (For the past years I've been managing project, rather than directly working in code.)

I searched my local repository, found the thing you asked to be fixed, fixed, and committed my fix. I have no idea whether that fix will now be seen in this PR, or if there is something additional I need to do to get it into the PR.